packages/extension-api/src/extension-api.d.ts

```diff
@@ -4373,7 +4374,7 @@ declare module '@podman-desktop/api' {
     };
 
     updateVersion(version: CliToolUpdateOptions): void;
-    onDidUpdateVersion: Event<string>;
+    onDidUpdateVersion: Event<string | undefined>;
```

Comment:
I'm wondering, should we receive an update event or a uninstall event ?

Reply:
updated